icicle/appUtils/keccak/keccak.cu

```diff
@@ -0,0 +1,275 @@
+#include "keccak.cuh"
+
+namespace keccak {
+#define ROTL64(x, y) (((x) << (y)) | ((x) >> (64 - (y))))
```

Comment:
what is the benefit of using #define for these functions and not just making them device functions?

Reply:
this is just for inlining